app/src/components/inputs/inputFilter/inputFilter.jsx

```diff
@@ -99,7 +111,22 @@ export class InputFilter extends Component {
   handleClickClear = () => this.props.onQuickClear();
 
   handleApply = () => {
-    this.props.tracking.trackEvent(this.props.eventsInfo.applyBtn);
+    if (this.props.eventsInfo.CLICK_APPLY_BUTTON) {
+      const currentAppliedFilters = getAppliedFilters(this.props.filterEntities);
+
+      const analyticsData = Object.keys(currentAppliedFilters)
+        .filter((key) => this.prevAppliedFilters.current[key] !== currentAppliedFilters[key])
+        .join('#');
+
+      if (analyticsData) {
+        this.props.tracking.trackEvent(this.props.eventsInfo.CLICK_APPLY_BUTTON(analyticsData));
+      }
+
+      this.prevAppliedFilters.current = currentAppliedFilters;
+    } else {
+      this.props.tracking.trackEvent(this.props.eventsInfo.applyBtn);
+    }
```

Comment:
Can we track these events using callbacks provided to the component?
We can pass them through the `EventsToolbar` (move `track` wrapper to it as well) during render.

Reply:
Yes we can, but looks like we need to rewrite not only EventsToolbar but `ProjectsToolbal` `UsersToolbar`